### PR TITLE
Created static UI of find shipment page (#1zhjyr7)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
find-shipment-desktop

find-shipment-mobile

File: src/views/FindShipment.vue [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+                        SKU
+                        <p>{{ $t("Color") }} : color</p>
+                        <p>{{ $t("Size") }}: size</p>
+                      </ion-label>
+                    </ion-item>
+                  </div>
+
+                  <div>
+                    <ion-chip outline>
+                      <!-- TODO -->
```
Comment (dt2patel) [inline src/views/FindShipment.vue]: what is the todo?

File: src/views/FindShipment.vue [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                      <ion-icon :icon="pricetag" />
+                      <ion-label>External ID</ion-label>
+                    </ion-chip>
+                  </div>
+
+                  <div>
+                    <ion-item lines="none">
+                      <ion-note slot="end">ATP Ordered</ion-note>
```
Comment (dt2patel) [inline src/views/FindShipment.vue]: these are two separate things, not one note

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/FindShipment.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/FindShipment.vue]: > what is the todo?

We have discussed earlier that some other icon or image we will use here and for now I should use pricetag tag so I have wrote TODO here.

Comment (dt2patel) [reply src/views/FindShipment.vue]: Add that to the code then so other contributors know

Comment (azkyakhan) [reply src/views/FindShipment.vue]: Done sir

File: src/views/FindShipment.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/FindShipment.vue]: Ok sir, I have updated the markup.
localhost_8100_find-purchase-order

File: src/views/FindPurchaseOrder.vue [hunk 3]
```diff
@@ -228,6 +227,14 @@ export default {
 </script>
 
 <style scoped>
+.items-metadata {
+  text-align: end;
+}
+
+.items-metadata :first-child {
```
Comment (dt2patel) [inline src/views/FindPurchaseOrder.vue]: why just the first child?

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/FindPurchaseOrder.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/FindPurchaseOrder.vue]: I wanted my first note to take the full width available and push another note in the next line, that's why I had applied on the first note only. There was no need to apply a display block to another note. 
But now I have applied it to both of them.

File: src/views/FindPurchaseOrder.vue [hunk 4]
```diff
@@ -123,16 +123,15 @@
...
                       <ion-label>Shopify ID</ion-label>
                     </ion-chip>
                   </div>
 
-                  <div>
-                    <ion-item lines="none">
-                      <ion-note slot="end">ATP Ordered</ion-note>
-                    </ion-item>
+                  <div class="items-metadata">
+                    <ion-note slot="end">ATP</ion-note>
```
Comment (dt2patel) [inline src/views/FindPurchaseOrder.vue]: if these ion-notes are in a div, slot end isn't doing anything right?

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/FindPurchaseOrder.vue (hunk 4 above)
Comment (azkyakhan) [reply src/views/FindPurchaseOrder.vue]: Yes sir

**Review by dt2patel — APPROVED** [review]

**Pull request merged**